flow/scripts/util.tcl

```diff
@@ -145,6 +145,9 @@ proc find_macros { } {
 }
 
 proc erase_non_stage_variables { stage_name } {
+  if { [info exists ::env(KEEP_VARS)] && $::env(KEEP_VARS) == 1 } {
+    return
+  }
```

Comment:
@maliberty I'm not sure if this is the best solution to prevent having the non-stage variables being deleted on the single-run. I'm open to suggestions here.

Reply:
I think this simple and works well enough.  The other option would be to capture all envars at the start and restore them between stages in floorplan_to_place.tcl